packages/core/expressions/src/components/RequestCard.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <div :class="['request-card', { active: props.active }]">
+    <CloseIcon
+      class="close-btn"
+      size="16"
+      @click="emit('remove')"
+    />
+    <div class="badges">
+      <KBadge
+        appearance="info"
+        class="protocol"
+      >
+        {{ protocol }}
+      </KBadge>
+      <KBadge
+        v-if="['http', 'https'].includes(protocol) && method"
+        :appearance="getMethodBadgeAppearance(method)"
+      >
+        {{ method?.toUpperCase() }}
```

Comment:
In the above `v-if` we have guaranteed `method` exists:
```html
v-if="['http', 'https'].includes(protocol) && method"
```

So here we don't need `?`:
```suggestion
        {{ method.toUpperCase() }}
```